WooCommerce/Classes/ViewRelated/Orders/Cells/OrderTableViewCell.swift

```diff
@@ -108,6 +107,19 @@ private extension OrderTableViewCell {
         paymentStatusLabel.layer.borderColor = borderColor
     }
 
+    /// For example, #560 Pamela Nguyen
+    ///
+    func title(for order: Order) -> String {
+        if let billingAddress = order.billingAddress {
+            return "#\(order.number) \(billingAddress.firstName) \(billingAddress.lastName)"
```

Comment:
similar to another comment on `"\(billingAddress.firstName) \(billingAddress.lastName)"`, let's localize this string or reuse the string with the name from billing address? in certain languages (like Mandarin), the name is in the inverse order and without space in between (`"\(billingAddress.lastName)\(billingAddress.firstName)"`) 🙂

Reply:
I didn't know that! I chose to localize it inside this class for now. 7b5c380b4a0b66371662d52aa59b38c531916714

Localizing in `Address.fullName` seems to affect other places. 😬 I'm also concerned about localizing with just spaces in them. This would be generated in `Localizable.strings` as:

```
﻿/* In Order List, the pattern to show the order number and the full name. For example, “#123 John Doe”. The %1$@ is the order number. The %2$@ is the first name. The %3$@ is the last name. */
"#%1$@ %2$@ %3$@" = "#%1$@ %2$@ %3$@";
```
If we end up with the same pattern in another place of the app, we might incorrectly translate one of them. I think if this happens twice, we might need a helper like the [`replaceOccurrences()`](https://github.com/woocommerce/woocommerce-ios/blob/09151699b550f89dd0c15d92906213e02b389105/WooCommerce/Classes/Extensions/NSMutableAttributedString%2BHelpers.swift#L17-L28) to use a custom pattern instead.

Reply:
looking at [the existing `Address.fullName`](https://github.com/woocommerce/woocommerce-ios/blob/86dcbe87419b0aa2ddffb32d8c1ec91bb8b9fe83/WooCommerce/Classes/Model/Address+Woo.swift#L12-L14), we have the same issue somewhere else:

https://github.com/woocommerce/woocommerce-ios/blob/86dcbe87419b0aa2ddffb32d8c1ec91bb8b9fe83/WooCommerce/Classes/Model/Address+Woo.swift#L12-L14

I think it'd be nice to localize the string in the extension above, and localize the order number + full name like`"#%1$@ %2$@"`. it is interesting how we localize the app right now by its value, is it possible to give different values to the key and value given our localization process? it feels like we could easily run into issues where we want to localize strings of the same value but in different contexts.

any reasons you don't think we should change this for all `fullName` use cases? maybe to limit the scope of changes? we currently use the billing address's `fullName` in 3 cases - order details, billing screen (order details > view billing information), and fulfill screen (order details > begin fulfillment). feel free to just change it for this PR though!

Reply:
> any reasons you don't think we should change this for all fullName use cases? maybe to limit the scope of changes?

Yup, it was mainly to limit the scope. As you mentioned, that `Address.fullName` is used in a lot of places and I wanted to avoid creating a regression with this PR. 😅 I created an issue to follow up with this in the future. https://github.com/woocommerce/woocommerce-ios/issues/2314

> is it possible to give different values to the key and value given our localization process?

I don't believe so. Our `Localizable.strings` is generated by [`localize.py`](https://github.com/woocommerce/woocommerce-ios/blob/develop/Scripts/localize.py) which uses the `NSLocalizedString` calls themselves. So there's no opportunity to create custom keys. The easiest solution to avoid the issue would be the pattern matching that I mentioned above. The larger alternative solution would be to go with something like SwiftGen's `Localizable.strings` generation. I kind of like that more. 

>  it feels like we could easily run into issues where we want to localize strings of the same value but in different contexts.

Yup. That's definitely something we should watch out for.